src/lib/configDataSeeder.js

```diff
@@ -0,0 +1,134 @@
+ http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, the Mojaloop files are distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ Contributors
+ --------------
+ This is the official list of the Mojaloop project contributors for this file.
+ Names of the original copyright holders (individuals or organizations)
+ should be listed with a '*' in the first column. People who have
+ contributed from an organization can be listed under the organization
+ that actually holds the copyright for their contributions (see the
+ Gates Foundation organization for an example). Those individuals should have
+ their names indented and be marked with a '-'. Email address can be added
+ optionally within square brackets <email>.
+ * Gates Foundation
+ - Name Surname <name.surname@gatesfoundation.com>
+
+ * Claudio Viola <claudio.viola@modusbox.com>
+ --------------
+******/
+'use strict'
+
+// const LedgerAccountTypeModel = require('../models/ledgerAccountType/ledgerAccountType')
+const ErrorHandler = require('@mojaloop/central-services-error-handling')
+const Config = require('./config')
+const SettlementService = require('../domain/settlement')
+const SettlementModel = require('../models/settlement/settlementModel')
+
+// const LedgerAccountTypesService = require('../domain/ledgerAccountTypes')
+const Db = require('./db')
+
+const SETTLEMENT_MODELS_ALIASES = {
+  CGS: {
+    name: 'CGS',
+    settlementGranularity: 'GROSS',
+    settlementInterchange: 'BILATERAL',
+    settlementDelay: 'IMMEDIATE',
+    requireLiquidityCheck: true,
+    ledgerAccountType: 'POSITION',
+    autoPositionReset: false,
+    settlementAccountType: 'SETTLEMENT'
+  },
+  DEFERREDNET: {
+    name: 'DEFERREDNET',
+    settlementGranularity: 'NET',
+    settlementInterchange: 'MULTILATERAL',
+    settlementDelay: 'DEFERRED',
+    requireLiquidityCheck: true,
+    ledgerAccountType: 'POSITION',
+    autoPositionReset: true,
+    settlementAccountType: 'SETTLEMENT'
+  },
+  INTERCHANGEFEE: {
+    name: 'INTERCHANGEFEE',
+    settlementGranularity: 'NET',
+    settlementInterchange: 'MULTILATERAL',
+    settlementDelay: 'DEFERRED',
+    requireLiquidityCheck: false,
+    ledgerAccountType: 'INTERCHANGE_FEE',
+    autoPositionReset: true,
+    settlementAccountType: 'INTERCHANGE_FEE_SETTLEMENT'
+  }
+}
+
+/**
+ * [initializeSeedData Adds configurable seeds data]
+ */
+async function initializeSeedData () {
+  const knex = Db.getKnex()
+  return knex.transaction(async trx => {
+    try {
+      // await initializeLedgerAccountTypes(trx)
+      await initializeSettlementModels(trx)
+
+      await trx.commit
+    } catch (err) {
+      await trx.rollback
+      throw ErrorHandler.Factory.reformatFSPIOPError(err)
+    }
+  })
+}
+
+async function initializeSettlementModels (trx) {
```

Comment:
This type of function feels more like it belongs in config instead of here... just my $0.02

Reply:
it's kinda config.. data.. seeder :D
I am not sure where would suit it better...